Sources/MapboxNavigation/NavigationMapView+VanishingRouteLine.swift

```diff
@@ -393,7 +396,7 @@ extension NavigationMapView {
         let baseColor: UIColor
         if let _ = congestionFeatures {
             overriddenLineLayerColor = lineLayerColorIfPresent(from: route)
-            baseColor = overriddenLineLayerColor ?? (isMain ? .trafficUnknown : .alternativeTrafficUnknown)
+            baseColor = overriddenLineLayerColor ?? (isMain ? trafficUnknownColor : alternativeTrafficUnknownColor)
```

Comment:
It should use the traffic color from `NavigationMapView` which supports customization.

Reply:
If we weren’t previously honoring the style’s custom traffic colors here, the changelog should mention that this issue has been fixed.